doc/manuals/Substitute.am

```diff
@@ -17,6 +17,15 @@ SUBSTITUTE=sed "\
 	s%@SYSCONFDIR@%$(sysconfdir)%g;\
 	"
 
-## Example of how to substitute:
-## squid.8: $(srcdir)/squid.8.in Makefile
-##	$(SUBSTITUTE) < $(srcdir)/squid.8.in > $@
+## Important:
+##  Due to use of $(SUBSTITUTE) the built files
+##  need to explicitly depend on Makefile at the
+##  location where they are built. The .md file
+##  does not need to be listed as a dependency.
+##  For Example, see how squid.8 is built in
+##  src/Makefile.am.
+##
+.md.8:
+	$(SUBSTITUTE) < $(srcdir)/`basename $<` | \
+	$(PANDOC) -t man -f gfm -s -o $@
```

Comment:
This command depends on PANDOC, but the whole `.md.8` rule is not protected by ENABLE_PANDOC_DOC. Should it be protected?

Reply:
Fixed in b12b817f3f